test/recipes/15-test_ecparam.t

```diff
@@ -18,8 +18,8 @@ use OpenSSL::Test::Utils;
 
 setup("test_ecparam");
 
-plan skip_all => "EC or EC2M isn't supported in this build"
-    if disabled("ec") || disabled("ec2m");
+plan skip_all => "EC, EC2M or SM2 isn't supported in this build"
+    if disabled("ec") || disabled("ec2m") || disabled("sm2");
```

Comment:
I don't think we should skip the non-SM2 tests in this suite just because SM2 is disabled.

Reply:
Fixed.